operations/helm/tests/build.sh

```diff
@@ -19,5 +19,5 @@ for FILEPATH in $TESTS; do
   TEST_NAME=$(basename -s '.yaml' "$FILEPATH")
 
   echo "Templating $TEST_NAME"
-  helm template "${TEST_NAME}" ${CHART_PATH} -f "${FILEPATH}" --output-dir "operations/helm/tests/${TEST_NAME}-generated"
+  helm template "${TEST_NAME}" ${CHART_PATH} -f "${FILEPATH}" --output-dir "operations/helm/tests/${TEST_NAME}-generated" --namespace default
```

Comment:
I think people would use mimir/GEM in a dedicated namespace in production. I'd rather we used some non standard name like "citestns" or something weird to grep easily. WDYT?

Reply:
I dont have a preference. I guess it doesnt hurt to have a non-standard namespace